index.html

```diff
@@ -17,14 +17,15 @@
           </div>
```

Comment:
- [ ] Let's make sure to update our `.github/workflows/linters.yml` to include `ESLint` we need to set [this config](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in our project.
- [ ] to update GitHub checks we need to update `.github/workflows/linters.yml` (for online checks)
- [ ] to update Local checks to include ESLint we need to add the [.eslintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json) and run this script 
```shell
npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
```
As mentioned in the configuration README.md file

Reply:
Thank you, I did that